lib/src/main/java/com/diffplug/spotless/json/JacksonJsonConfig.java

```diff
@@ -27,6 +27,8 @@ public class JacksonJsonConfig extends JacksonConfig {
 
 	protected Map<String, Boolean> jsonFeatureToToggle = new LinkedHashMap<>();
```

Comment:
should this be renamed jsonGeneratorFeatureToggle for clarity ?

Reply:
This is `protected`, I'd rather not change the ABI, not a big deal, I prefer to keep this as-is.